Pod/Classes/JSSAlertView.swift

```diff
@@ -181,19 +181,19 @@ public class JSSAlertView: UIViewController {
 		let titleString = titleLabel.text! as NSString
 		let titleAttr = [NSFontAttributeName:titleLabel.font]
```

Comment:
Hello there @huangboju, shouldn't this be: `let titleAttr = [NSFontAttributeName:titleLabel.font!] as [String: Any]`?